### PR TITLE
skip test case secureboot_enable_disable when VBS is enabled

### PR DESCRIPTION
**Pull request by 123lzxm**
Signed-off-by: Yanan Shen <yanans@vmware.com>

### CODE REVIEW

File: windows/secureboot_enable_disable/secureboot_enable_disable.yml [hunk 1]
```diff
@@ -31,6 +31,15 @@
...
           when: vm_firmware is undefined or vm_firmware | lower != 'efi'
 
+        - name: "Get VBS status of VM"
+          include_tasks: ../../common/vm_get_vbs_status.yml
+        - name: "Skip {{ current_testcase_name }} with VBS enabled"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip test case due to secure boot cannot be disabled for VM with VBS enabled."
+            skip_reason: "Not Applicable"
+          when: vm_vbs_enabled is defined and vm_vbs_enabled | bool
```
Comment (Tomorrow9) [inline windows/secureboot_enable_disable/secureboot_enable_disable.yml:41]: `vm_vbs_enabled` is defined in task `vm_get_vbs_status.yml`

File: windows/secureboot_enable_disable/secureboot_enable_disable.yml [hunk 2]
```diff
@@ -31,6 +31,15 @@
             skip_reason: "Not Applicable"
           when: vm_firmware is undefined or vm_firmware | lower != 'efi'
 
+        - name: "Get VBS status of VM"
```
Comment (Tomorrow9) [inline windows/secureboot_enable_disable/secureboot_enable_disable.yml:34]: Can add a when condiditon "guest_os_ansible_architecture == "64-bit"" for this block.

File: windows/secureboot_enable_disable/secureboot_enable_disable.yml (hunk 1 above)
Comment (123lzxm) [reply windows/secureboot_enable_disable/secureboot_enable_disable.yml:41]: Yes, the first conditon can be removed.

File: windows/secureboot_enable_disable/secureboot_enable_disable.yml (hunk 2 above)
Comment (123lzxm) [reply windows/secureboot_enable_disable/secureboot_enable_disable.yml:34]: Ok, thanks.

**Review by Tomorrow9 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**